src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -32,6 +32,11 @@ abstract class KernelTestCase extends TestCase
      */
     protected static $kernel;
 
+    private function doTearDown()
```

Comment:
Maybe it doesn't bother too many people, but the visibility of this method has been changed from v4.3.3:

https://github.com/symfony/symfony/blob/dd4d355126184444fb3f940d17f83a9462f15be9/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L40